### PR TITLE
Add component type definitions (#2)

### PR DESCRIPTION
**Pull request by giladsegal**
### 🔦 Summary
...

### ✅ Checklist
- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: src/Calendar/Calendar.uni.driver.d.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  getFocusedDayElement: () => Promise<HTMLElement | null>;
+  pressLeftArrow: () => Promise<void>;
+  pressRightArrow: () => Promise<void>;
+  getSelectedDay: () => Promise<string>;
+  getWidth: () => Promise<CSSStyleDeclaration['width']>;
+  triggerKeyDown: (params: SyntheticEventData) => Promise<void>;
+  isFocusedDayVisuallyUnfocused: () => Promise<boolean>;
+  containsVisuallyUnfocusedDay: () => Promise<boolean>;
+  isTwoMonthsLayout: () => Promise<boolean>;
+  getMonthDropdownDriver: () => Promise<DropdownLayoutDriver>;
```
Comment (mosheKerbel) [inline src/Calendar/Calendar.uni.driver.d.ts]: should be DropdownLayoutUniDriver

File: src/Calendar/Calendar.uni.driver.d.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  getYearDropdownDriver: () => Promise<DropdownLayoutDriver>;
```
Comment (mosheKerbel) [inline src/Calendar/Calendar.uni.driver.d.ts]: should be DropdownLayoutUniDriver

File: src/Calendar/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  | 'pl'
+  | 'it'
+  | 'ru'
+  | 'ja'
+  | 'ko'
+  | 'tr'
+  | 'sv'
+  | 'no'
+  | 'nl'
+  | 'da';
```
Comment (mosheKerbel) [inline src/Calendar/index.d.ts]: missing these:
      'zh',
      'th',
      'cs',
https://github.com/wix/wix-style-react/blob/8d37e4d0dc991a90a784a874a375b08dfa2b24ab/src/Calendar/Calendar.js#L419

**Comment by mosheKerbel** [comment]
We are currently not exporting any of the uniDrivers, but the types here are only for the *.uni and not for specific platforms drivers. Is this intended? @shlomitc

**Comment by shlomitc** [comment]
@mosheKerbel - I think I talked with Gilad about creating only for the unidrivers, but actually, we should have for both if we want maximum value from it. @giladsegal - do you remember if this was the conclusion we had?

File: src/Calendar/Calendar.uni.driver.d.ts (hunk 1 above)
Comment (giladsegal) [reply src/Calendar/Calendar.uni.driver.d.ts]: I will check all of these tomorrow, I was OOO for a few days,
After I will finish, we can merge it

**Pull request closed without merging**